scripts/mixpanel.js

```diff
@@ -0,0 +1,69 @@
+var MIXPANEL_CUSTOM_LIB_URL = '/js/mixpanel/mixpanel-2-latest.min.js'
```

Comment:
Does this file exist on `docs.salad.com`?

Reply:
no, I didn't find the connected mixpanel script on docs.salad.com

Reply:
Let's replace the URL with `https://salad.com/js/mixpanel/mixpanel-2-latest.min.js`.

Additionally, will you copy-paste the tracking code back in and _not_ format it with Prettier?